gammapy/catalog/tests/test_hawc.py

```diff
@@ -0,0 +1,23 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import pytest
+from ..hawc import SourceCatalog2HWC
+
+@pytest.fixture #TODO: add here a scope like in test_gammacat.py?
```

Comment:
Yes, add session scope here. No need to read the files multiple times during a test run.